### DISCUSSION TITLE
Possible to add autobrr to services?

### DISCUSSION THREAD

**Discussion by SSamiK** [seerr-team/seerr#630] [Ideas]
Is it possible to add [autobrr ](https://github.com/autobrr/autobrr) as a service?

**Comment by fallenbagel** [comment]
Nope

**Comment by SSamiK** [comment]
Fair enough... Really like to use autobrr over sonarr/radarr as those are a bit big and cumbersome. But I'll manage.

**Comment by Ondjultomte** [comment]
Any possibility on this integration ?
